packages/fxa-settings/src/pages/InlineRecoveryKeySetup/container.tsx

```diff
@@ -12,7 +12,7 @@ export const InlineRecoveryKeySetupContainer = ({
 }: {
   integration: Integration;
 } & RouteComponentProps) => {
-  const [currentStep, setCurrentStep] = useState<number>(1);
+  const [currentStep] = useState<number>(1);
```

Comment:
The set state function will be needed to make this functional in `FXA-10079`, might as well leave it in ;)